### PR TITLE
Fix for issue #65

### PR DESCRIPTION
**Pull request by alkedr**
Related to #65

### CODE REVIEW

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java [hunk 1]
```diff
@@ -68,23 +70,31 @@ protected void fillElement(WebElement element, Object value) {
...
+            element.sendKeys(getClearTextInputElementCharSequence(element) + value.toString());
         }
     }
 
+    // Returns sequence of backspaces and deletes that will clear element
+    // element.clear() can't be used because clear() generates separate onchange event
+    // element must be <textarea> or <input> with type text, password, email etc
+    // See https://github.com/yandex-qatools/htmlelements/issues/65
+    private static String getClearTextInputElementCharSequence(WebElement element) {
+        return StringUtils.repeat(Keys.DELETE.toString() + Keys.BACK_SPACE, element.getText().length());
```
Comment (emaks) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java]: for some fields ``` element.getText() ``` will return empty string when  ``` element.getAttribute("value") ``` will return not empty result

Comment (alkedr) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java]: We need to replace `element.getText().length()` with `Math.max(element.getText().length(), element.getAttribute("value"))`.

**Comment by aik099** [comment]
Aren't there any universal method for getting element value?

**Comment by alkedr** [comment]
I thought `element.getText()` was universal method for getting element value.

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java (hunk 1 above)
Comment (aik099) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java]: Ah. I finally see, why there is a problem. In the original code, that was used to create that PR: https://github.com/minkphp/MinkSelenium2Driver/blob/master/src/Selenium2Driver.php#L659-L662 we're:
1. operation only on `input` and `textarea` elements
2. always taking the `value` attribute value (the `attribute` method in selenium in fact looks in JS `.value` property on DOM node and gets it's value, rather then HTML attribute `value`, that doesn't exist on textarea)

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java [hunk 2]
```diff
@@ -68,23 +70,31 @@ protected void fillElement(WebElement element, Object value) {
...
         } else if (isSelect(element)) {
             Select select = new Select(element);
             select.selectByValue(value.toString());
         } else if (isTextArea(element)) {
-            element.sendKeys(value.toString());
+            element.sendKeys(getClearTextInputElementCharSequence(element) + value.toString());
         }
     }
 
+    // Returns sequence of backspaces and deletes that will clear element
```
Comment (lanwen) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java]: for future prs :) - use javadoc style comments on methods - `\** */`

**Pull request merged**